content/articles/dealing-with-python's-missing-values-a-complete-guide/index.md

```diff
@@ -0,0 +1,105 @@
+ Machine learning and deep learning frameworks have nothing to do with data purification. As part of pre-processing, we'll look into Data Cleaning. The dataset's data structure can be improved by removing errors, duplication, corrupted items, and other issues.

```

Comment:
A more briefer introduction could have been given.

Talk about what are missing values & preprocessing? why do we need to handle them? how to deal with it? what if we don't? Any live examples?

Have a look at other Section articles, to get an idea.

Reply:
A single line explanation of each would be enough for the intro